### ISSUE TITLE
Difference between Raw logs and Range

### ISSUE THREAD

**Issue by Heyzi** [VictoriaMetrics/victorialogs-datasource#196]
victorialogs-datasource: v0.13.2
victorialogs: v1.4.0
I want to plot the number of errors at each point in time, but I get strange results in the Raw logs and Range. 
If this is correct behavior, where is the difference described?
What query should I write to make the graph look more like the Raw logs graph?

<img width="1335" alt="image" src="https://github.com/user-attachments/assets/f97aab45-b78a-42f2-9b90-bc457002ff33" />
<img width="1335" alt="image" src="https://github.com/user-attachments/assets/0a64e612-6782-4ada-bff1-9010336ab8c9" />
<img width="1335" alt="image" src="https://github.com/user-attachments/assets/dd2d69ec-e7eb-4fdd-bc4e-7965217fe8b1" />

**Comment by tiny-pangolin** [comment]
You can make the timeseries panel more like the logs volume panel, changing the graph style to bar under Graph styles when you are editing the panel
I believe the logs volume panel in explore doesn't use a normal panel that, and it is not exposed to the user for making their own panels.

**Comment by leshcat** [comment]
Have similar question, how do you use time range/instant types?
Are there any examples or documentation related to actual usage of those handlers?

I am currently trying to convert Loki PromQL:
`sum(count_over_time({env="dev", container="myapp"} |= `[ERROR]` [1h])) by (env, container)`

which gives following graph:

<img width="995" alt="Image" src="https://github.com/user-attachments/assets/8b952230-2432-4760-9e96-5134b85565cd" />

to a VLogs query and get similar graph.

VLogs query i came up is:
`_time:1h {container="myapp", env="dev"} "[ERROR]" | stats count()`

but it only works correctly on Instant type and only as a dot:

<img width="1040" alt="Image" src="https://github.com/user-attachments/assets/f550ef52-5f2c-4cf3-a1ab-ec2674c00b3f" />

**Comment by tiny-pangolin** [comment]
this can be solved by adding  `by(_time:5m)` in the query which will give you give you the count in an hour window so the whole query would be `_time:1h {container="myapp", env="dev"} "[ERROR]" | stats by (_time:5m) count()`

**Comment by tiny-pangolin** [comment]
try remove ing `_time:1h` from the start of your victorialogs query, and change the `by(_time:5m)` to `by(_time:1h)` and see if that works. _time:1h is limiting the response to messages sent in the past hour instead of getting the duration from Grafana and the _time in the by should match the duration for count_over_time in your loki query

**Comment by leshcat** [comment]
> try remove ing `_time:1h` from the start of your victorialogs query, and change the `by(_time:5m)` to `by(_time:1h)` and see if that works. _time:1h is limiting the response to messages sent in the past hour instead of getting the duration from Grafana and the _time in the by should match the duration for count_over_time in your loki query

Makes sense, but unfortunately that didn't do the trick and values are now weird:

<img width="1127" alt="Image" src="https://github.com/user-attachments/assets/2734a781-983d-4e44-91d4-5c07b5b6e4c0" />

**Comment by dmitryk-dk** [comment]
Hi @Heyzi , @leshcat ! Can you check the issue on the [latest](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.15.0-victorialogs) version of the victorialogs. Is it reproducible?

**Comment by tommysitehost** [comment]
![Image](https://github.com/user-attachments/assets/cbb468da-3de0-4525-bb14-83c9e26f65ef)

still happening on the latest version of VictoriaLogs (victoriametrics/victoria-logs:v1.17.0-victorialogs) by the look of it
